.gitattributes

```diff
@@ -67,3 +49,33 @@ Programs/test_frozenmain.h  linguist-generated=true
 *.html  diff=html
 *.py    diff=python
 *.md    diff=markdown
+
+# Generated files
+# https://github.com/github/linguist/blob/master/docs/overrides.md
+#
+# To always hide generated files in local diffs, mark them as binary:
+# $ git config diff.generated.binary true
+#
+**/clinic/*.h                       linguist-generated=true diff=generated
+*_db.h                              linguist-generated=true diff=generated
+Doc/library/token-list.inc          linguist-generated=true diff=generated
+Include/internal/pycore_ast.h       linguist-generated=true diff=generated
+Include/internal/pycore_ast_state.h linguist-generated=true diff=generated
+Include/opcode.h                    linguist-generated=true diff=generated
+Include/token.h                     linguist-generated=true diff=generated
+Lib/keyword.py                      linguist-generated=true diff=generated
+Lib/token.py                        linguist-generated=true diff=generated
+Modules/config.c                    linguist-generated=true diff=generated
+Objects/typeslots.inc               linguist-generated=true diff=generated
+Parser/parser.c                     linguist-generated=true diff=generated
+Parser/token.c                      linguist-generated=true diff=generated
+Programs/test_frozenmain.h          linguist-generated=true diff=generated
+Python/Python-ast.c                 linguist-generated=true diff=generated
+Python/deepfreeze/*.c               linguist-generated=true diff=generated
+Python/frozen_modules/*.h           linguist-generated=true diff=generated
+Python/frozen_modules/MANIFEST      linguist-generated=true diff=generated
+Python/opcode_targets.h             linguist-generated=true diff=generated
+Python/stdlib_module_names.h        linguist-generated=true diff=generated
+Tools/peg_generator/pegen/grammar_parser.py linguist-generated=true diff=generated
+aclocal.m4                          linguist-generated=true diff=generated
+configure                           linguist-generated=true diff=generated
```

Comment:
Does `linguist` follow `[attr]` declarations in `.gitattributes`?  If so, it might be nicer to add `[attr]generated linguist-generated diff=generated`, and then we just need to tag the generated files with `generated` rather than the full `linguist-generated=true diff=generated`.  That also makes it just one place to change things if we later want to add another tag to generated files.

Reply:
Oooooh, that's a nice thought. I'll check if that is possible.

Reply:
Tested and verified on my fork. Thanks! 🍰 🥳 🎉

Reply:
(I'll update the no-EOL and CRLF sections with macros in a separate PR.)